pJanks/jeopardy
# Player icons and Yoda Message

### What's this PR do?
- Refactor styling on player icons and avatars
- Add ".active-player-style" class which will be applied to either: `.player-1-container` or `.player-2-container` or `.player-3-container`
- Add `messageFromYoda: (message) =>` function where we can pass in: `Correct, you are!` or `Incorrect, you are!` or `End of round 1` etc.
### Where should the reviewer start?
Open web app and click through different screens
### How should this be manually tested?
Try selecting different answers and entering player names to confirm layouts are appropriate
### Potential changes
May need to refactor sizes
